ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/README.md

```diff
@@ -0,0 +1,180 @@
+:scrollbar:
+:data-uri:
+:toc2:
+:linkattrs:
+
```

Comment:
I don't think you need those four lines.

Reply:
Removed